sumpy/kernel.py

```diff
@@ -239,17 +239,7 @@ def postprocess_at_target(self, expr, bvec):
         The typical use of this function is to apply target-variable
         derivatives to the kernel.
         """
-        from sumpy.tools import (ExprDerivativeTaker,
-            DifferentiatedExprDerivativeTaker)
-        expr_dict = {(0,)*self.dim: 1}
-        expr_dict = self.get_derivative_transformation_at_target(expr_dict)
-        if isinstance(expr, ExprDerivativeTaker):
-            return DifferentiatedExprDerivativeTaker(expr, expr_dict)
-
-        result = 0
-        for mi, coeff in expr_dict.items():
-            result += coeff * self._diff(expr, bvec, mi)
-        return result
+        return expr
```

Comment:
This is strange to me... wouldn't this default implementation risk invalidly skipping derivatives?